articles/Angle/agEUR_post_Euler.md

```diff
@@ -0,0 +1,217 @@
+
+## Lessons learned and a path forward for V2
+
+On April 5th, 2023, Angle published [Euler Hack Reflection: Learning, Adapting, and Strengthening Angle Protocol](https://www.angle.money/#/blog/announcements/euler-hack-reflection-learning-adapting-and-strengthening-angle-protocol) which give a glimpse of a possible way forward for the Angle Protocol. 
+
+The ongoing discussion surrounding the protocol's future includes the introduction of a Price Stability Module 2.0. This proposed module would support agEUR by backing it with a diversified basket of various Euro-denominated stablecoins and financial instruments. This enhanced system would be adept at managing stablecoin depegs. It would operate in tandem with the Borrowing module and algorithmic market operations. As part of this plan, the protocol would divest itself of USDC reserves, with additional details to be disclosed shortly.
+
+In light of these observations, a discussion has been initiated on the Angle Governance Forum ([Ideas for re-launching the protocol - thread](https://gov.angle.money/t/ideas-for-re-launching-the-protocol/674)) to explore options for safely reopening the protocol after the hack while repayments are still pending, as well as refining the Angle Core module design. The community is encouraged to actively participate in these discussions, sharing insights and collaborating to build a more secure and robust future for the Angle Protocol.
+
+The recent pause of the Core Module presents an opportune moment to address existing technical debt, paving the way for a more resilient, robust, and scalable system. Before the Euler hack, the Core Module's limitations were exposed when a rapid decline in USDC's price led to the liquidation of most hedging agents. This left the protocol inadequately hedged and resulted in a significant decrease in the protocol surplus. The absence of effective hedging mechanisms within the Core Module underscores the need for a system to withstand such events and manage unfavorable USD/EUR price fluctuations. Enhancements should ensure agEUR price stability and foster a more resilient and scalable protocol.
+
+On April 14th 2023, further precisions were provided on potential way forward for Angle V2 throught the governance forum thread [Angle Protocol - Core Module V2](https://gov.angle.money/t/angle-protocol-core-module-v2/). Key requirements for the future system include low slippage for mints and burns, scalability, autonomy during unforeseen events, equal treatment of stablecoin holders, ability to accept Euro-denominated assets, proper risk diversification, and no exposure to foreign exchange risk.
+
+## Risk assessment
```

Comment:
I think pablo had mentioned about preparing payloads for emergency situations ahead of time, cant remember off top of my head if he said anything else relevant to this section?